taxi/views.py

```diff
@@ -87,3 +102,21 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    form_class = DriverCreateForm
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+class DriverDeleteView(LoginRequiredMixin, generic.DeleteView):
+    model = Driver
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+class DriverLicenseUpdateView(LoginRequiredMixin, generic.UpdateView):
+    model = Driver
+    field = ["license_number"]
+    form_class = DriverLicenseUpdateForm
+    template_name = "taxi/license_form.html"
```

Comment:
you could use just `DriverLicenseUpdateForm` without additional template for `license_form` and make everything in driver_form using `object|yesno`

Reply:
Thank you for this idea. I will implement this a little bit later ;)